### PR TITLE
Rel 4.13.0.3

### PR DESCRIPTION
**Pull request by Sakoram**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
 
+## [v4.13.0.3] - 2021-01-29
```
Comment (miczyg1) [inline CHANGELOG.md:16]: `2021-01-27` we plan the release on that day.

**Comment by miczyg1** [comment]
@Sakoram tried to run built binaries on apu1 and apu2?

**Comment by Sakoram** [comment]
> @Sakoram tried to run built binaries on apu1 and apu2?

Yes, they boots.

File: CHANGELOG.md (hunk 1 above)
Comment (Sakoram) [reply CHANGELOG.md]: fixed

**Comment by miczyg1** [comment]
@Sakoram great. I will merge it now and place a tag tomorrow

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**